2020/scoring/clean_and_join_csvs.sh

```diff
@@ -19,6 +20,7 @@ UC_OUT=../results/Unsat_Core_Track.csv
 
 SQ_ORIG_IN=single-query/Job_info_orig.csv
 SQ_2019_IN=single-query/Job_info_2019.csv
+SQ_PATCH=SQ-wrong-sat-result.csv
```

Comment:
This requires a file from origin/master which was added after the branching for the pull request happened.

Reply:
I suggest we fix this problem when rebasing to master.